research-hub-web/src/app/components/search-results/subhubs.component.ts

```diff
@@ -0,0 +1,104 @@
+import { Component, OnInit, Type } from '@angular/core';
```

Comment:
is this supposed to be under search-results? is it just temporary?

Reply:
oh yeah i think was a drag and drop mistake